src/installer/pkg/projects/Microsoft.DotNet.ILCompiler/Microsoft.DotNet.ILCompiler.pkgproj

```diff
@@ -0,0 +1,44 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <GeneratePackage>true</GeneratePackage>
+    <IncludeSymbolsInPackage>true</IncludeSymbolsInPackage>
+    <PackageDescription>Provides a native AOT compiler and runtime for .NET</PackageDescription>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- For now, send symbols package into a dummy directory to avoid conflicts with dotnet/runtime and
+         to avoid issues with objwriter not having the rigth ELF version resource -->
+    <SymbolPackageOutputPath>$([MSBuild]::NormalizeDirectory('$(ArtifactsPackagesDir)', 'Ignored'))</SymbolPackageOutputPath>
+  </PropertyGroup>
+
+  <Target Name="GetIlcCompilerFiles"
+          DependsOnTargets="ResolveLibrariesFromLocalBuild"
+          BeforeTargets="GetPackageFiles">
+
+    <ItemGroup Condition="'$(PackageTargetRuntime)' != ''">
+      <File Include="@(LibrariesRuntimeFiles)">
+        <TargetPath>framework</TargetPath>
```

Comment:
Small stiling nit, this kind of metadata makes the file smaller and simpler to read if declared inline: i.e:

```xml
<File Include="@(LibrariesRuntimeFiles)" TargetPath="framework" />
```